parcel_test.go

```diff
@@ -35,9 +35,8 @@ func TestAddGetDelete(t *testing.T) {
 	// prepare
 	// настройте подключение к БД
 	db, err := sql.Open("sqlite", "tracker.db")
-	if err != nil {
-		require.NoError(t, err)
-	}
+
+	require.ErrorIs(t, err, nil)
```

Comment:
В комментарии про ошибки я перепутал строчки, так что он неверный, нужно так делать

```suggestion
	require.NoError(t, err)
```